backend/src/server.ts

```diff
@@ -36,7 +38,27 @@ app.listen({ port: PORT, host: IP }, (err) => {
     app.log.error(err);
     process.exit(1); // eslint-disable-line
   }
+  const caPaths = [
```

Comment:
It would be helpful for future eyes if we added  small comment on why these files are passed along.

`/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt` is required for security communication with internal services

`/etc/pki/tls/certs/odh-ca-bundle.crt` & `/etc/pki/tls/certs/odh-trusted-ca-bundle.crt` are self-signed trusted CA cert bundles managed by the ODH operator

Reply:
Additionally, do you need a comment on why you needed to add these file contents manually `https.globalAgent.options.ca` below?

Reply:
I think that's a great idea, we are exploring other implementations cause this is a really explicit way of just adding the certs and we might wanna be more flexible in the futures, I'm gonna add some comments to give more context